Dockerfile

```diff
@@ -1,7 +1,40 @@
-FROM domerd/marine-core:marine
+ARG MARINE_CORE_TAG=marine
 
-RUN python3.8 -m pip install tox
+FROM domerd/marine-core:$MARINE_CORE_TAG as builder
 
 WORKDIR /marine-python
 
-CMD tox
+COPY setup* ./
```

Comment:
This stage will pass even if you have no `setup.py`, because of the wildcard

Reply:
It will work as long as there's a file starting with `setup`